app/controllers/order/goods_details.js

```diff
@@ -11,6 +11,10 @@ export default Ember.Controller.extend({
   otherDetails: "",
   sortProperties: ["id"],
   sortedGcRequests: Ember.computed.sort("model.goodcityRequests", "sortProperties"),
+  gcRequestNull: Ember.computed("model.goodcityRequests", function() {
```

Comment:
May be rename to hasGcRequest?

Also correct Indentation

Reply:
Already done, was committing same.